### PR TITLE
Change subscription format

### PR DESCRIPTION
**Pull request by fgalan**
Implements issue #2014 

@crbrox 
@kzangeli 
@fgalan

### CODE REVIEW

**Comment by fgalan** [comment]
LGTM (passing the ball to other reviewers)

File: src/lib/jsonParseV2/parseSubscription.cpp [hunk 1]
```diff
@@ -230,6 +230,24 @@ std::string parseSubscription(ConnectionInfo* ciP, ParseData* parseDataP, JsonDe
...
+    if (!throttling.IsInt64())
+    {
+      alarmMgr.badInput(clientIp, "throttling is not a int");
+      OrionError oe(SccBadRequest, "throttling is not an int");
+
+      return oe.render(ciP, "");
+    }
+    destination->throttling.seconds = throttling.GetInt64();
+  }
+  else if (!update) // throttling was not set and it is update
```
Comment (kzangeli) [inline src/lib/jsonParseV2/parseSubscription.cpp]: throttling was not set and it is **not** update  ?

File: src/lib/jsonParseV2/parseSubscription.cpp [hunk 2]
```diff
@@ -416,17 +434,34 @@ static std::string parseNotification(ConnectionInfo* ciP, SubscribeContextReques
...
+      OrionError oe(SccBadRequest, "url http notification is missing");
 
       return oe.render(ciP, "");
     }
-    scrP->reference.string = callback.GetString();
+    const Value& url = http["url"];
+    if (!url.IsString())
+    {
+      alarmMgr.badInput(clientIp, "url http notification is not an string");
+      OrionError oe(SccBadRequest, "url http notification is not an string");
```
Comment (kzangeli) [inline src/lib/jsonParseV2/parseSubscription.cpp]: type: an => a

File: src/lib/jsonParseV2/parseSubscription.cpp (hunk 1 above)
Comment (fgalan) [inline src/lib/jsonParseV2/parseSubscription.cpp]: Fixed in 8b64d66706ad5d1fd8fcd6a305d0e731150361c2

File: src/lib/jsonParseV2/parseSubscription.cpp (hunk 2 above)
Comment (fgalan) [inline src/lib/jsonParseV2/parseSubscription.cpp]: Fixed in 8b64d66706ad5d1fd8fcd6a305d0e731150361c2

File: src/lib/jsonParseV2/parseSubscription.cpp [hunk 3]
```diff
@@ -230,6 +230,24 @@ std::string parseSubscription(ConnectionInfo* ciP, ParseData* parseDataP, JsonDe
     destination->status = statusString;
   }
 
+  // Throttling
+  if (document.HasMember("throttling"))
+  {
+    const Value& throttling = document["throttling"];
+    if (!throttling.IsInt64())
+    {
+      alarmMgr.badInput(clientIp, "throttling is not a int");
```
Comment (crbrox) [inline src/lib/jsonParseV2/parseSubscription.cpp]: my fault ... `an int`

Comment (fgalan) [inline src/lib/jsonParseV2/parseSubscription.cpp]: Fixed in 1e72e60

**Comment by kzangeli** [comment]
LGTM

**Comment by crbrox** [comment]
LGTM

**Pull request merged**